frontend/src/components/modals/NewPageModal.vue

```diff
@@ -69,6 +71,11 @@ async function createPage() {
       name: name.value,
       parentId: parent.value,
     });
+
+    if (updateProjectPagesCallback) {
+      updateProjectPagesCallback(await useInternalApi<HangarProjectPage[]>(`pages/list/${props.projectId}`, false, "get"));
```

Comment:
if you try catch one, it makes sense to do it for the other too :D

can make it more concise by using `await useInternalApi().catch(() => handleRequestError)` instead

Reply:
```ts
if (updateProjectPages) {
  const newPages = await useInternalApi<HangarProjectPage[]>(`pages/list/${props.project.id}`, false, "get").catch((e) => handleRequestError(e, ctx, i18n))
  if (newPages) updateProjectPages(newPages)
}
```

like this ? The other one is already part of an outer try-catch for creating the page itself.

Reply:
oh, thats fine then